force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -101,17 +127,18 @@ public with sharing class ClassesObjectsMethods {
      * @return The number of vowels in the string.
      */
     public static Integer countVowels(String str) {
+        
         // Initialize the result Integer
+        Integer vowelCount = 0;
+        List<String> letters = str.split('');
 
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+        for (String letter : letters) {
+            if (letter.containsAny('aeiou')) {
```

Comment:
You can also put the vowels in a list and ask if the letter is in the list

Reply:
Good point, made that change.